### PR TITLE
avoid having to escape twice in go/vt/vtgate/plugin_mysql_server.go file

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

**Comment by tanjunchen** [comment]
@bbeaudreault

**Comment by tanjunchen** [comment]
@deepthi

**Comment by tanjunchen** [comment]
@morgo

**Comment by tanjunchen** [comment]
@systay

File: go/vt/vtgate/plugin_mysql_server.go [hunk 1]
```diff
@@ -118,7 +118,7 @@ func (vh *vtgateHandler) ConnectionClosed(c *mysql.Conn) {
 }
 
 // Regexp to extract parent span id over the sql query
-var r = regexp.MustCompile("/\\*VT_SPAN_CONTEXT=(.*)\\*/")
+var r = regexp.MustCompile(`/\\*VT_SPAN_CONTEXT=(.*)\\*/`)
```
Comment (systay) [inline go/vt/vtgate/plugin_mysql_server.go]: Nice find! The un-escaped string should be:
```
`/\*VT_SPAN_CONTEXT=(.*)\*/`
```

Comment (tanjunchen) [reply go/vt/vtgate/plugin_mysql_server.go]: OK Thank you

**Comment by tanjunchen** [comment]
@sougou @systay   the go.mod file is a problem once the go.mod file is pulled. Do you think ?

**Pull request merged**